### PR TITLE
bugfix 254 - timeout on resource level

### PR DESCRIPTION
**Pull request by venkatavivek-ntnx**

### CODE REVIEW

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test

>Acceptance test run status: failure
>

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test

>Acceptance test run status: failure
>

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by siddharth-nutanix** [comment]
/ok-to-test
>Acceptance test run status: success
>Line code coverage is 60.6

File: nutanix/resource_nutanix_network_security_rule.go [hunk 1]
```diff
@@ -44,6 +43,12 @@ func resourceNutanixNetworkSecurityRule() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"wait_timeout": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      DEFAULTWAITTIMEOUT,
+				ValidateFunc: validation.IntAtLeast(MINIMUMWAITTIMEOUT),
+			},
```
Comment (shreevari) [inline nutanix/resource_nutanix_network_security_rule.go]: Should this be changed to `Timeouts` attribute?

Comment (venkatavivek-ntnx) [reply nutanix/resource_nutanix_network_security_rule.go]: Yeah. I missed that. Now it's done.

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test

>Acceptance test run status: failure
>

**Review by shreevari — APPROVED** [review]

**Comment by venkatavivek-ntnx** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Comment by siddharth-nutanix** [comment]
/ok-to-test
>Acceptance test run status: failure
>

**Review by siddharth-nutanix — APPROVED** [review]

**Review by siddharth-nutanix — APPROVED** [review]

**Pull request merged**